internal/tlsutil/tls.go

```diff
@@ -24,8 +24,27 @@ import (
 	"github.com/pkg/errors"
 )
 
-// NewClientTLS returns tls.Config appropriate for client auth.
-func NewClientTLS(certFile, keyFile, caFile string) (*tls.Config, error) {
+// Validate and return TLS renegotiation settings
+func GetRenegotiation(option string) (strategy tls.RenegotiationSupport, err error) {
+	switch option {
```

Comment:
same here RE: making these fields case-insensitive. In general, if a flag accepts an enumerated string, it's a good idea to cast user flag input to lowercase so that `NEVER`, `never` and `Never` are all valid input.

Reply:
see above